xml/security-security_features.xml

```diff
@@ -6,15 +6,43 @@
  <!ENTITY % entities SYSTEM "entity-decl.ent"> %entities;
 ]>
 <chapter xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xlink="http://www.w3.org/1999/xlink" xml:id="newsecurity">
- <title>&productnamereg;: Security Features Overview</title>
+ <title>&productnamereg;: Security Planning and Features</title>
+ <section xml:id="idg-all-security-security_planning-xml-1">
+  <title>Security Planning</title>
+  <para>
+   &productnamereg; is a complex system which has many options which
```

Comment:
*that* has many options (will read better)

Reply:
done in https://github.com/SUSE-Cloud/doc-cloud/pull/950